db/dcrsqlite/sqlite.go

```diff
@@ -16,6 +16,8 @@ import (
 	apitypes "github.com/decred/dcrdata/api/types"
 	"github.com/decred/dcrdata/blockdata"
 	_ "github.com/mattn/go-sqlite3" // register sqlite driver with database/sql
+	"os"
```

Comment:
as with other PR, segregate the std lib imports.  Put them above github stuff with a newline between the std lib and github imports